db/migrate/20150128185746_seed_new_storytime_actions_and_permissions.rb

```diff
@@ -0,0 +1,9 @@
+class SeedNewStorytimeActionsAndPermissions < ActiveRecord::Migration
+  def up
+    Storytime::Action.seed
+    Storytime::Permission.seed
+  end
```

Comment:
Pretty sure that was a migration pulled in from the Dev branch for the additional "Manage Email Subscriptions" action and permission.